go/teams/hidden_loader_test.go

```diff
@@ -170,13 +165,13 @@ func testHiddenLoadFailsIfServerDoesntCommitLinks(t *testing.T) bool {
 	// make the architect run
```

Comment:
maybe remove this function altogether?

Reply:
maybe when we get rid of the architect from keybase/web code?